components/script_injector/common/mojom/BUILD.gn

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//mojo/public/tools/bindings/mojom.gni")
+
+mojom("mojom") {
+  generate_java = true
```

Comment:
I would actually not generate java for this because I don't think we should ever be calling this directly from java code